.github/workflows/ci.yml

```diff
@@ -97,9 +91,9 @@ jobs:
       run: |
         sudo rm -f /etc/apt/sources.list.d/dotnetdev.list /etc/apt/sources.list.d/microsoft-prod.list
         sudo apt update
-        sudo apt install postgresql-12
```

Comment:
is there a reason to downgrade?

Reply:
yeh for some reason 1804 apt only has v10 available (I checked locally in a docker container)